### PR TITLE
[2.13] jbang + HQL queries backports

### PR DESCRIPTION
**Pull request by pjgg**
### Summary

[Add coverage for new lines in HQL queries](https://github.com/quarkus-qe/quarkus-test-suite/pull/1056) fedinskiy
[Make jbang use the same repository as maven](https://github.com/quarkus-qe/quarkus-test-suite/pull/1058) fedinskiy

Please select the relevant options.
- [X] Backport

### CODE REVIEW

**Comment by fedinskiy** [comment]
a) There is a CI failure, I suppose we also need to backport a fix for Windows CI
b) I suggest to wait for https://github.com/quarkus-qe/quarkus-test-suite/pull/1064 to be merged and backport it as well

**Review by fedinskiy — CHANGES_REQUESTED** [review]
See above

**Comment by pjgg** [comment]
depends on: https://github.com/quarkus-qe/quarkus-test-suite/pull/1083

File: sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/hibernate/reactive/AbstractDatabaseHibernateReactiveIT.java [hunk 1]
```diff
@@ -287,5 +290,15 @@ public void ensureSessionIsPropagatedOnReactiveTransactions() {
                 .then().statusCode(HttpStatus.SC_CREATED);
     }
 
+    @Test
+    @Disabled("https://github.com/quarkusio/quarkus/issues/31117")
+    public void newLineInQuery() {
+        Response author = getApp().given()
+                .get("/library/by-author/Dlugi");
+        System.out.println(author.body().asString());
```
Comment (fedinskiy) [inline sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/hibernate/reactive/AbstractDatabaseHibernateReactiveIT.java:298]: Do we need this output?

**Review by fedinskiy — CHANGES_REQUESTED** [review]

**Pull request merged**